chef/cookbooks/crowbar/recipes/prepare-upgrade-scripts.rb

```diff
@@ -308,10 +308,13 @@
   only_if { nova_node }
 end
 
-template "/usr/sbin/crowbar-run-nova-online-migrations.sh" do
-  source "crowbar-run-nova-online-migrations.sh.erb"
+template "/usr/sbin/crowbar-migrations-after-upgrade.sh" do
+  source "crowbar-migrations-after-upgrade.sh.erb"
   mode "0775"
   owner "root"
   group "root"
+  variables(
+    heat_server: roles.include?("heat-server")
+  )
   only_if { roles.include?("nova-controller") }
```

Comment:
what if we have heat-server on a node which doesn't have nova-controller?

Reply:
Hm, I did not expect it , but it probably could happen.

So I guess I should just check if heat is deployed (and still call the action from nova controller to make it simpler)